homework1_aboutme_styles.css

```diff
@@ -0,0 +1,41 @@
+body {
+	font-family: helvetica;
```

Comment:
Nice use of inheritance here to have your font family rule be used by the entire page.

Something to look at for next time - you can declare multiple values for font-family like this:
     font-family: helvetica, arial, sans-serif;

Doing that basically means that if the browser cannot find the first font, it will try to use the next font.  The "sans-serif" at the end will tell the browser to just use whatever default sans-serif font it can find if all else fails.  (There is also a "serif" keyword you can use for the default serif font.)

Realistically, any computer should have helvetica, so this is a non-issue in this case.